static/skywire-hypervisor.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
Can we rename this directory (`/static`) to `/init`?

Reply:
I don't see why not. The structure of the repo is up to you and Rudi. I chose static because that is where the systemd services were in skybian.

Reply:
done